site/fr/tutorials/quickstart/advanced.ipynb

```diff
@@ -0,0 +1,417 @@
+      "version": "3.7.3"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "rX8mhOLljYeM"
+      },
+      "source": [
+        "##### Copyright 2019 Les auteurs de TensorFlow."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "cellView": "form",
+        "colab_type": "code",
+        "id": "BZSlp3DAjdYf",
+        "colab": {}
+      },
+      "source": [
+        "#@title Sous licence Apache, Version 2.0 (la \"Licence\");\n",
+        "# vous ne pouvez pas utiliser ce fichier, sauf conformément avec la licence.\n",
+        "# Vous pouvez obtenir une copie de la Licence sur\n",
+        "#\n",
+        "# http://www.apache.org/licenses/LICENSE-2.0\n",
+        "#\n",
+        "# Sauf si requis par la loi en vigueur ou par accord écrit, le logiciel\n",
+        "# distribué sous la licence est distribué \"TEL QUEL\", SANS GARANTIE NI\n",
+        "# CONDITION DE QUELQUE NATURE QUE CE SOIT, implicite ou explicite.\n",
+        "# Consultez la Licence pour connaître la terminologie spécifique régissant\n",
+        "# les autorisations et les limites prévues par la Licence."
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "3wF5wszaj97Y"
+      },
+      "source": [
+        "# TensorFlow 2 démarrage rapide pour avancés"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "DUNzJc4jTj6G"
+      },
+      "source": [
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://www.tensorflow.org/tutorials/quickstart/advanced\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />Voir sur TensorFlow.org</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/tutorials/quickstart/advanced.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Exécuter dans Google Colab</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/en/tutorials/quickstart/advanced.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />Voir la source sur GitHub</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/en/tutorials/quickstart/advanced.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />Télécharger le notebook</a>\n",
+        "  </td>\n",
+        "</table>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "hiH7AC-NTniF"
+      },
+      "source": [
```

Comment:
We need to add a text cell with the community translations disclaimer.
Can take this from the [fr/README](https://github.com/tensorflow/docs/blob/master/site/fr/README.md):

```
Note: Notre communauté `TensorFlow` a traduit ces documents. 
Étant donné que les traductions effectuées dans la communauté sont plus optimales, 
rien ne garantit qu'il s'agit d'un reflet exact et à jour de la documentation anglaise officielle. 
Si vous avez des suggestions pour améliorer cette traduction, 
veuillez envoyer une demande d'extraction ou `pull request`) à [tensorflow/docs](https://github.com/tensorflow/docs). 
Pour vous porter volontaire pour écrire ou réviser les traductions de la communauté, 
contactez la [liste docs@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs).
```